en/guide/general_usage.md

```diff
@@ -20,10 +20,19 @@ DroneCore APIs do not raise exceptions! Instead, methods that can fail return su
 
 The various classes also all provide methods getting human readable strings from their associated enum (e.g. [DroneCore::connection_result_str()](../api_reference/classdronecore_1_1_drone_core.md#classdronecore_1_1_drone_core_1a84c40dcefcafe888c38a5ed8dd93b0af), [Telemetry::result_str()](../api_reference/classdronecore_1_1_telemetry.md#classdronecore_1_1_telemetry_1a05c6355b7f8743250b2a7a611ea5fb4a)). You can see how these are used in the example code.
 
+

```

Comment:
@julianoes FYI, I added this "general" note that vehicles potentially having multiple command sources. Of course monitoring flight modes alone isn't a complete solution, but "better than nothing".